### ISSUE TITLE
Add new sport: open water swimming

### ISSUE THREAD

**Issue by xmgz** [SamR1/FitTrackee#400]
as suggested, I open this issue to discuss about **swimming** activities

as _FitTrackee_ requires a valid .gpx file from **outdoor activities** I request **Open Water Swimming** to be added as activity.

Adding a generic **Swimming** would make possible to manually add an activity recorded in the swimming pool (without gps data) so it would be **prefered** (in my opinion). Same as other indoor activities without .gpx values.

this is a .gpx file from garmin device if you need to check if there's any diference to be taken in consideration 
[activity_11528282494(1).zip](https://github.com/SamR1/FitTrackee/files/12076201/activity_11528282494.1.zip)  so others could upload here a file from other devices to check if there is any particular format or values to consider.

I would suggest that _elevation values_ should be overwritten to 0 (zero) in case GPS glitches or errors could introduce any value (please comment if this is incorrect from my side)

Please comment so _devs_ can add swimming in future releases. **Thank you**.

**Comment by SamR1** [comment]
Hi,

Thanks for opening this issue and the .gpx file.
With the current version of **FitTrackee**, it displays the following data from the file:  
![Screenshot 2023-07-18 at 08-42-03 FitTrackee - Séance](https://github.com/SamR1/FitTrackee/assets/3525167/1177f04a-e8fa-4571-83be-a998b720872a)

It shouldn't display ascent and descent, since the file does not contain elevation data (no `<ele>` element) .
```xml
      <trkpt lat="<LATITUTE>" lon="<LONGITUDE>">
        <time>2023-07-10T10:37:45.000Z</time>
        <extensions>
          <ns3:TrackPointExtension>
            <ns3:atemp>20.0</ns3:atemp>
            <ns3:cad>0</ns3:cad>
          </ns3:TrackPointExtension>
        </extensions>
      </trkpt>
```
 
I'll fix that and add **open water swimming** (indoor activities need more change and different display).

**Comment by xmgz** [comment]
Speed values usually are noted as *minutes per one hundred meters* (**min/100m**), at least in the two or three apps/sites I have used.

As conversion is trivial, I don't know if speed charts and values are the same for every sport or if they can be formatted by the kind of activity.

I'm totally fine if speed keeps the current format and I don't know if _min/100m_ is actually any kind of standard.

Thank you.

_Note_: this could be applicable also to running where we usually talk about _pace_ in terms of _minutes per kilometer (mile)_.

So initally could be best to keep it as it is and then.. open a diferent issue

**Comment by SamR1** [comment]
Hi,

Yes, I agree. I've already opened an issue to display pace (https://github.com/SamR1/FitTrackee/issues/302) :).

**Comment by SamR1** [comment]
released in v0.7.20
